lib/content-types/merge.js

```diff
@@ -155,6 +168,34 @@ const squish = (types) => {
         return plugin;
       });
 
+      if (mergedType.identifier) {
```

Comment:
Or, at the very least, happen twice. Once when it's loaded, and again when it's been merged